rosdep/base.yaml

```diff
@@ -4994,6 +4994,11 @@ r-base-dev:
   fedora: [R-devel]
   gentoo: [dev-lang/R]
   ubuntu: [r-base-dev]
+range-v3:
+  debian: [librange-v3-dev]
```

Comment:
This package is only available for debian buster and later.  We still support debian stretch for Melodic, so this key will have to be more specific.  See https://github.com/ros/rosdistro/blob/1cf2a0266f6abaaf86293772cb3f3570a2b8bf82/rosdep/base.yaml#L229 for an example on how to do this.

Reply:
Marked as unavailable for stretch. The package is available in stretch-backports repository, but I guess that doesn't count, right?

Reply:
Correct, we can only depend on things in the main, security, universe, and multiverse repositories.